### DISCUSSION TITLE
Feature: hide albums without a cover on Browse - Album tab

### DISCUSSION THREAD

**Discussion by rare-magma** [jcorporation/myMPD#828] [Ideas]
I was wondering if it would make sense to add an option to hide albums from the Browser view that don't have a cover / fail to load.
I'm asking this because when I use the Browse tab and select "Albums" I'd like to see only albums with a cover, otherwise I'd use the other filters that return a table / list.

Almost half of the albums fail to load in my case:
![image](https://user-images.githubusercontent.com/45106055/187526666-d4efd901-a3ea-45d4-930c-a84414699bf1.png)

**Comment by jcorporation** [comment]
At the time of creating the result myMPD does not know if a cover is available.
